Sage/carbon
# feat(portrait, profile): describe portrait and profile using design tokens, update tests

### Proposed behaviour

Describes profile and portrait components using design tokens.
Removes unused `theme` props.
Fixes icon rendering size.
In `Profiles` describes the size of the `name` and `email` using tokens.

### Current behaviour

Portrait and Profile components use the theme styled-component property.
Theme is now passed as props.
Icons have a fixed size regardless of the portrait size.
Name and email in the Profile is represented by px.

### Checklist




#### QA


### Additional context

Portrait:
![image](https://user-images.githubusercontent.com/44143630/151977717-4f67981f-3d68-48b7-b397-50ccb44e19da.png)
![image](https://user-images.githubusercontent.com/44143630/151977778-75b5989f-4bda-4893-b67d-51f727b0fda0.png)
Profile:
![image](https://user-images.githubusercontent.com/44143630/151977947-8b4966e5-fdb6-4241-9409-49be9465d938.png)

### Testing instructions

The correct appearance can be checked in the components: Portrait and Profile.
For theme: mint and aegean there should be no differences.
Use devtools to check if components use css variables.
